data_diff/database.py

```diff
@@ -671,6 +825,10 @@ class Snowflake(Database):
         "TIMESTAMP_LTZ": Timestamp,
         "TIMESTAMP_TZ": TimestampTZ,
     }
+    NUMERIC_TYPES = {
+        "NUMBER": Decimal,
+        "FLOAT": Float,
```

Comment:
Snowflake has a ton of aliases, but they are all 4-bytes. I suspect that just works, because `numeric_precision` is just always 53 or whatever? 👍🏻 

<img width="378" alt="CleanShot 2022-06-21 at 09 55 35@2x" src="https://user-images.githubusercontent.com/97400/174817154-2b962656-cf4b-430d-a1bb-a11aa0eccfff.png">

Reply:
These aliases are usually "normalized" to a single type by the dbs before stored in INFORMATION_SCHEMA.